packages/library/index.js

```diff
@@ -139,11 +139,18 @@ module.exports = (neutrino, opts = {}) => {
         neutrino.use(loaderMerge('lint', 'eslint'), { envs: ['browser', 'commonjs'] });
       }
     })
-    .when(process.env.NODE_ENV === 'production', () => {
+    .when(process.env.NODE_ENV === 'production', (config) => {
       neutrino.use(minify);
-      neutrino.config
-        .plugin('module-concat')
-          .use(optimize.ModuleConcatenationPlugin);
+      config.plugin('module-concat').use(optimize.ModuleConcatenationPlugin);
+
+      if (neutrino.options.debug) {
```

Comment:
I think we'd want to do this regardless of being in `production`, right? If they are using `debug`, we should also use these I think...

Reply:
Sure, we can remove the conditional. I'll send a patch now.

Reply:
Sorry, I think you may have misread my comment. We want these options to be set always when using debug, not always using production. It needs to come out of this `when` and go into its own when using `debug`.

Reply:
I definitely misread the comment. Sorry for the confusion.

Reply:
No worries at all. 😃